src/coffee/client.coffee

```diff
@@ -1,6 +1,7 @@
 _ = require 'underscore'
 Rest = require './connect/rest'
 TaskQueue = require './task-queue'
+AuthenticationService    = require './services/authentication'
```

Comment:
I actually wonder why the API endpoint was named like this, since it's related to customer...